### PR TITLE
New packages: rbw-1.4.3, rofi-rbw-1.0.0

### LINKED ISSUE #28973: Package Request: rbw
Description: rbw is a unofficial command line client for Bitwarden.
 
Official repo: https://git.tozt.net/rbw/about/
GitHub mirror: https://github.com/doy/rbw

### PR DESCRIPTION
**Pull request by ram02z**
#### General
- [x] This is a new package and it conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements)

#### Have the results of the proposed changes been tested?
- [x] I use the packages affected by the proposed changes on a regular basis and confirm this PR works for me
- [ ] I generally don't use the affected packages but briefly tested this PR

#### Does it build and run successfully? 
(Please choose at least one native build and, if supported, at least one cross build. More are better.)
- [x] I built this PR locally for my native architecture, (x86_64-musl)
- [x] I built this PR locally for these architectures (if supported. mark crossbuilds):
  - [x] aarch64-musl
  - [x] armv7l
  - [x] armv7l-musl
  - [x] armv6l
  - [x] armv6l-musl

Closes #28973

#### Note:
Not sure whether to include the shell scripts in the [bin](https://git.tozt.net/rbw/tree/bin) directory, as they aren't installed on [Arch](https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=rbw) and are behind feature flags on [Nix](https://github.com/NixOS/nixpkgs/blob/master/pkgs/tools/security/rbw/default.nix#L77).

### CODE REVIEW

**Comment by ram02z** [comment]
Turns out not every arch has the same build directory, so I am letting `cargo` run it instead.

**Comment by ram02z** [comment]
Hmm, turns out rbw doesn't support the arm architecture, but it should support aarch64 according to [this](https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=rbw#n9). 

I will remove `arm*` from the archs list on the next push, but regarding aarch64 support I would appreciate some guidance on what to do next.

File: srcpkgs/rbw/template [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+homepage="https://git.tozt.net/rbw"
+distfiles="${homepage}/snapshot/${pkgname}-${version}.tar.gz"
+checksum=7255e322b8c6bd6ee9da00f4c54211e94a132abec15b9f51b8351af31a4744ac
+
+do_build() {
+	cargo build --release --locked
+
+	cargo run --release --locked --bin rbw -- gen-completions bash > rbw.bash
+	cargo run --release --locked --bin rbw -- gen-completions fish > rbw.fish
+	cargo run --release --locked --bin rbw -- gen-completions zsh  > rbw.zsh
```
Comment (paper42) [inline srcpkgs/rbw/template:19]: This is probably the problem with cross compilation (arm*, aarch64*). `cargo run` compiles for the target architecture (arm) and then tries to run the arm binary on the host computer (x86_64) which fails.

File: srcpkgs/rbw/template [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+pkgname=rbw
+version=1.3.0
+revision=1
+build_style=cargo
+depends="pinentry"
+short_desc="Unofficial command line client for Bitwarden"
+maintainer="Omar Zeghouani <omarzeghouanii@gmail.com>"
+license="MIT"
+homepage="https://git.tozt.net/rbw"
+distfiles="${homepage}/snapshot/${pkgname}-${version}.tar.gz"
```
Comment (paper42) [inline srcpkgs/rbw/template:11]: please don't use $homepage and $pkgname

**Comment by ram02z** [comment]
Ok, so its still failing but atleast its not my template at fault.

**Comment by ram02z** [comment]
Since new packages take a while to get merged, I added `rofi-rbw`, which I was planning to add after the original PR got merged anyway.

**Comment by TinfoilSubmarine** [comment]
Tested `rbw` package on x86_64, LGTM.

File: srcpkgs/rofi-rbw/template [hunk 3]
```diff
@@ -0,0 +1,19 @@
+# Template file for 'rofi-rbw'
+pkgname=rofi-rbw
+version=0.5.0
+revision=1
+build_style=python3-pep517
+make_install_target="rofi_rbw-${version}-py3-none-any.whl"
+hostmakedepends="python3-setuptools python3-wheel"
+depends="python3 python3-ConfigArgParse rbw"
```
Comment (classabbyamp) [inline srcpkgs/rofi-rbw/template:8]: needs `xdotool` (xorg) or `wtype` (wayland) to work properly

**Review by classabbyamp — COMMENTED** [review]
tested and works for me, aside from comment below

File: srcpkgs/rofi-rbw/template (hunk 3 above)
Comment (ram02z) [reply srcpkgs/rofi-rbw/template:8]: That's true, but should I put both as dependencies? I don't use x11, so xdotool is a pointless dependency for me.

Comment (Piraty) [reply srcpkgs/rofi-rbw/template:8]: no

**Comment by ram02z** [comment]
Rebuilt for Rust 1.61

**Comment by ram02z** [comment]
@paper42 Are the CI failures related to the age of the PR? I can open up a new PR if so.

**Comment by classabbyamp** [comment]
no, it was a transient issue with CI

also,you can do a force push to re-trigger CI

**Comment by classabbyamp** [comment]
rofi-rbw added manpage, but it is currently being installed in `/usr/lib/python3.10/site-packages/share/man/man1/rofi-rbw.1` instead of `/usr/share/man/man1/rofi-rbw.1`.

a patch to remove:

https://github.com/fdw/rofi-rbw/blob/3153097f85bfb6b0b0468c4b3dab7b5b1667b115/setup.cfg#L26-L27

and

```sh
vman src/rofi_rbw/docs/rofi-rbw.1
```
should work

**Comment by ram02z** [comment]
> vman src/rofi_rbw/docs/rofi-rbw.1

Thanks! Didn't know a manpage was added.

**Pull request merged**